### DISCUSSION TITLE
Play Structure > Progression Structure > Individual

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#72] [Framework Discussions]
# Play Structure > Progression Structure > Individual

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
This game progression outside matches consists of individual growth, unlocking new features and gamemodes after reaching specific Access Levels

**Comment by Tomathy-dev** [comment]
Players also have access to individual progression like leveling. Leveling a character is done usually through questing (completing quests in the world), while leveling a profession, which is also individual, is done through either gathering resources or transforming these resources into improved items.
